mrc/externals/diode.py

```diff
@@ -0,0 +1,45 @@
+import wiringpi
+
+
+class WS2811:
+    def __init__(self, rgb, spi_ch=1, spi_speed=2000000):
+        self._verify_color(rgb)
+        self.rgb = {
+            'r': rgb[0],
+            'g': rgb[1],
+            'b': rgb[2]
+        }
+        self.spi_ch = spi_ch
+        self.spi_speed = spi_speed
+
+    def __call__(self):
+        self.send_color_via_spi()
+
+    def __setitem__(self, key, value):
+        if type(value) is int and type in range(0, 256) and key in ['r,g,b']:
```

Comment:
I think it should be "value" instead of "type" in 
if ... and type in range(0, 256) ...

Also, range(0, 256) = range(256)

Reply:
Geez.